<launch>
   <node name="head_arduino" pkg="ros_homebot" type="head_node.py" output="screen">
      <param name="verbose" type="bool" value="0" />
   </node>
</launch>
